src/ray/core_worker/core_worker.cc

```diff
@@ -1309,6 +1309,26 @@ Status CoreWorker::CreatePlacementGroup(
   return Status::OK();
 }
 
+Status CoreWorker::RemovePlacementGroup(const PlacementGroupID &placement_group_id) {
+  std::shared_ptr<std::promise<void>> promise = std::make_shared<std::promise<void>>();
+  std::shared_ptr<Status> removal_status = std::make_shared<Status>(Status::OK());
+  // Synchronously wait for placement group removal.
+  RAY_UNUSED(gcs_client_->PlacementGroups().AsyncRemovePlacementGroup(
+      placement_group_id, [promise, removal_status](Status status) {
+        *removal_status = status;
+        promise->set_value();
+      }));
+  if (promise->get_future().wait_for(std::chrono::seconds(
+          RayConfig::instance().gcs_server_request_timeout_seconds())) !=
+      std::future_status::ready) {
+    std::ostringstream stream;
+    stream << "There was timeout in removing the placement group. It is probably "
```

Comment:
Use std::string directly?

Reply:
Actually, I added an << placement group id, so I will keep it.